doc/source/tutorial/stats/continuous_studentized_range.rst

```diff
@@ -0,0 +1,18 @@
+
+.. _continuous-studentized_range:
+
+Studentized Range Distribution
+==============================
+This distribution has two shape parameters :math:`\nu>0` and :math:`k>0` and support is :math:`q\geq0`.
+
+.. math::
+   :nowrap:
+
+    \begin{eqnarray*}
+        f(q;k,\nu) = \frac{\nu^{\nu/2}}{\Gamma(\nu/2)\,2^{\nu/2-1}}\,  \int_0^\infty\int_{-\infty}^\infty  s^{\nu-1}\,e^{-\nu s^{2}/2}  \,sk(k-1) \varphi(z) \varphi(sq+z) [\Phi(sq+z)-\Phi(z)]^{k-2} \,\mathrm{d}z \, \mathrm{d}s\\
+
+        F(q;k,\nu) = \frac{\nu^{\nu/2}}{\Gamma(\nu/2)\,2^{\nu/2-1}}\, \int_0^\infty\int_{-\infty}^\infty s^{\nu-1}\,e^{-\nu s^{2}/2}  k\varphi(z) [\Phi(sq+z)-\Phi(z)]^{k-1} \mathrm{d}z  \, \mathrm{d}s\\
```

Comment:
@DominicChm @swallan Would the one who didn't write this check carefully against the reference?

Please also include the nu = infinity approximation.

Reply:
That's me, I will check after vectorizing ttest trimmed tonight.